game/scripts/npc/abilities/spirit_breaker_greater_bash.txt

```diff
@@ -38,7 +38,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "damage"                                          "14 20 26 32 80 128"
+        "damage"                                          "14 20 26 32 44 56"
```

Comment:
Surely this is to much? "64 96" at OAA levels sounds more reasonable. This is still a huge nerf but "44 56" is too harsh. The hero will probably die if bash deals that little damage. You should still deal high damage if you go a high damage build.

Reply:
We agreed we don't want spirit breaker to be instant kill hero but be a crowd control hero.